src/index.js

```diff
@@ -0,0 +1,14 @@
+import './style.css';
+import { Tasks } from './task.js';
+
+const AddNewTask = document.querySelector('.Add');
+const tasks = [];
```

Comment:
- [ ] You have done great so far, :+1: :+1: But I think there are some requirements you still need to complete before moving to the next milestone.

> Create an index.js file and set an array of some simple to do tasks (array of objects). Each task object should contain three keys:
description [string].
completed [bool].
index: [number].

- -  According to the project requirement, you are meant to have an array of todo's that you are going to iterate over and display on that page once the page load, I can see you added the functionality to add a todo, but that's not a requirement for this milestone. I'll suggest you create an array of 3 - 4 todos, and them loop over them, and display them on the page once the page loads. :+1: :100: eg: `const tasks = [
  {
    description: 'An example description in string',
    completed: false,
    index: 10,
  },
];
`